src/components/view.js

```diff
@@ -49,6 +51,7 @@ export default {
           matched.instances[name] = undefined
         }
       }
+      data.props = resolveProps(route, component, matched.props && matched.props[name])
```

Comment:
Should merge with the original `data.props`, or it would be a breaking change as the current code disallows passing props to `<router-view>`

data.props = {...data.props, resolveProps(route, component, matched.props && matched.props[name])}